### DISCUSSION TITLE
Pls how can i get started contributing and make my first PR

### DISCUSSION THREAD

**Discussion by iamkennis** [EddieHubCommunity/support#555] [First-Timers]
Hi eddie, i will love to know how to get started in contributing in the community projects. thanks for the opportunity

**Comment by AllanRegush** [comment]
Eddie is pretty busy lately. So, I will try to answer. Contributing to opensource is not only code. You can help by creating issues for bugs and suggestions, creating documentation, and testing. We have a repository for first-timers. https://github.com/EddieJaoudeCommunity/hacktoberfest-practice you can make your first pull request there. 

If you have any other questions let me know. 🤓

**Reply by iamkennis** [reply]
Wow! Thanks

**Reply by eddiejaoude** [reply]
Sorry @iamkennis I am not ignoring you, I have been super busy with client deadlines. I am now going through GitHub notifications 🤓 Thank you @AllanRegush for the great reply 👍 

@iamkennis after you have gone through the workflow on the `hacktoberfest-practice` repo, have a look at our other repos and we can help you with any questions and getting started, just let us know - have a look out for issue labels with `good first issue` too
